File: src/main/java/com/fabianrodriguez/nohit/hispano/backend/entity/Continente.java
package com.fabianrodriguez.nohit.hispano.backend.entity;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

import java.io.Serializable;
import java.util.Date;

@Entity
@Table(schema = "public", name = "continente")
@AllArgsConstructor
@NoArgsConstructor
@Data
@ToString
public class Continente implements Serializable {

    @Id
    private Long id;

    @Column(name = "nombre")
    private String nombre;

    @Column(name = "fecha_insert")
    @Temporal(TemporalType.TIMESTAMP)
    private Date fechaInsert;

}

File: src/models/supabase/continente.py
class Continente:
    TABLE_NAME = "continente"

    def __init__(self, id, nombre, fecha_insert):
        self.id = id
        self.nombre = nombre
        self.fecha_insert = fecha_insert

    def __repr__(self):
        return f"Continente(id={self.id}, nombre={self.nombre}, fecha_insert={self.fecha_insert})"